erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -86,20 +97,16 @@ def update_so(self, so_item, picked_qty, item_code):
 				frappe.throw(
 					_(
 						"You are picking more than required quantity for {}. Check if there is any other pick list created for {}"
-					).format(item_code, so_doc.name)
+					).format(item_code, item.sales_order)
```

Comment:
Bug: in multi-uom cases, `picked_qty` is in stock UOM and `actual_qty` is in UOM specified in SO.
It always throws the error even though I am picking half qty
Eg: Item Double Dozen Eggs > Stock UOM: Nos > 1 Box = 24 Nos

Place SO for 10 Boxes (240 Nos) > Pick 5 Boxes (120 Nos) > It says I am overpicking